compose/service.py

```diff
@@ -1695,3 +1699,135 @@ def rewrite_build_path(path):
         path = WINDOWS_LONGPATH_PREFIX + os.path.normpath(path)
 
     return path
+
+
+def exec_build(path=None, tag=None, quiet=False, fileobj=None,
```

Comment:
`path=None` is following the original codebase.
https://github.com/docker/docker-py/blob/37e096f6add7e26ada3d6840ce9a9ce341bbdf23/docker/api/build.py#L16-L23

Reply:
The point is that in this API `path` is optional because `fileobj` can be used instead and `fileobj` is not used in your PR. Checked in https://github.com/docker/docker-py/blob/37e096f6add7e26ada3d6840ce9a9ce341bbdf23/docker/api/build.py#L125

Reply:
I understand it.
can i use incompatible signature?

Reply:
@ulyssessouza I fixed it